sql/pgwire_test.go

```diff
@@ -292,6 +292,9 @@ func TestPGPreparedQuery(t *testing.T) {
 		"SHOW database": {
 			base.Results(""),
 		},
+		"SELECT 1 FROM system.descriptor WHERE descriptor != $1": {
```

Comment:
Extremely indirectly, but it's done in the zone tests in CLI.

I've made this return the descriptor, which shook out the same bugs as the CLI tests did.